accounts/models.py

```diff
@@ -186,7 +186,7 @@ def can_create_study(self):
                 self.has_perm(
                     LabPermission.CREATE_LAB_ASSOCIATED_STUDY.codename, obj=lab
                 )
-                for lab in self.labs.values("id")
+                for lab in self.labs.all()
```

Comment:
ugh, my bad. Change to `only("id")` - keeps the query fast; we only ever need ID for permission checks